From 53a2140a2ff81e0c963b2bee92f26ce0848afa12 Mon Sep 17 00:00:00 2001
From: Will Thompson <will@willthompson.co.uk>
Date: Mon, 15 Jul 2019 15:57:05 +0100
Subject: [PATCH 6/6] appdata: add <release> for 3.32.1

---
 data/appdata/gnote.appdata.xml.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/appdata/gnote.appdata.xml.in b/data/appdata/gnote.appdata.xml.in
index 662740b2..0d443a14 100644
--- a/data/appdata/gnote.appdata.xml.in
+++ b/data/appdata/gnote.appdata.xml.in
@@ -22,6 +22,7 @@
  <update_contact>aurisc4_at_gmail.com</update_contact>
  <content_rating type="oars-1.1"/>
  <releases>
+  <release version="3.32.1" date="2019-04-14"/>
   <release version="3.32.0" date="2019-03-17"/>
  </releases>
  <translation type="gettext">gnote</translation>
-- 
2.20.1

