diff --git a/data/org.gnome.Gnote.appdata.xml.in b/data/org.gnome.Gnote.appdata.xml.in
index 570bdc9a..12ce38ac 100644
--- a/data/org.gnome.Gnote.appdata.xml.in
+++ b/data/org.gnome.Gnote.appdata.xml.in
@@ -50,14 +50,14 @@
       <li>Translation updates</li>
     </ul>
   </release>
-  <release version="40.rc" date="2021-03-13">
+  <release version="40~rc" date="2021-03-13">
     <ul>
       <li>Use reverse-domain convention for naming desktop and search provider files</li>
       <li>Refactor D-Bus support to reuse the connection established by GtkApplication</li>
       <li>Update AppData file format</li>
     </ul>
   </release>
-  <release version="40.beta" date="2021-02-21">
+  <release version="40~beta" date="2021-02-21">
     <ul>
       <li>Add accelerator Ctrl+D for insert timestamp</li>
       <li>WebDAV sync no longer uses wdfs</li>
@@ -65,7 +65,7 @@
       <li>Fix crash when synchronizing immediately after configuring</li>
     </ul>
   </release>
-  <release version="40.alpha" date="2021-01-10">
+  <release version="40~alpha" date="2021-01-10">
     <ul>
       <li>Change shortcut for find first/next to Ctrl-G/Ctrl-Shitf-G</li>
       <li>Change icons and remove labels from find next/previous</li>
