From 5f3390a61e608cf490901147014e37bfb0a85889 Mon Sep 17 00:00:00 2001
From: Will Thompson <will@willthompson.co.uk>
Date: Mon, 15 Jul 2019 15:55:28 +0100
Subject: [PATCH 4/6] appdata: add <release> for 3.32.0

---
 data/appdata/gnote.appdata.xml.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/data/appdata/gnote.appdata.xml.in b/data/appdata/gnote.appdata.xml.in
index 851a5926..47edb72e 100644
--- a/data/appdata/gnote.appdata.xml.in
+++ b/data/appdata/gnote.appdata.xml.in
@@ -21,5 +21,8 @@
  <url type="translate">https://wiki.gnome.org/TranslationProject</url>
  <update_contact>aurisc4_at_gmail.com</update_contact>
  <content_rating type="oars-1.1"/>
+ <releases>
+  <release version="3.32.0" date="2019-03-17"/>
+ </releases>
 </application>
 
-- 
2.20.1

