diff --git a/data/appdata/org.gnome.Gnote.appdata.xml.in b/data/appdata/org.gnome.Gnote.appdata.xml.in
index 15143c0a..e77d05b9 100644
--- a/data/appdata/org.gnome.Gnote.appdata.xml.in
+++ b/data/appdata/org.gnome.Gnote.appdata.xml.in
@@ -14,7 +14,7 @@
   <_p>Gnote also supports synchronization, making it simple to use it on multiple devices.</_p>
  </description>
  <screenshots>
-  <screenshot type="default" width="624" height="351">https://wiki.gnome.org/Apps/Gnote?action=AttachFile&amp;do=get&amp;target=gnote_main_window.png</screenshot>
+  <screenshot type="default">https://wiki.gnome.org/Apps/Gnote?action=AttachFile&amp;do=get&amp;target=gnote_main_window.png</screenshot>
  </screenshots>
  <url type="homepage">https://wiki.gnome.org/Gnote</url>
  <url type="bugtracker">https://gitlab.gnome.org/GNOME/gnote/issues</url>
@@ -45,14 +45,14 @@
       <li>Translation updates</li>
     </ul>
   </release>
-  <release version="40.rc" date="2021-03-13">
+  <release version="40~rc" date="2021-03-13">
     <ul>
       <li>Use reverse-domain convention for naming desktop and search provider files</li>
       <li>Refactor D-Bus support to reuse the connection established by GtkApplication</li>
       <li>Update AppData file format</li>
     </ul>
   </release>
-  <release version="40.beta" date="2021-02-21">
+  <release version="40~beta" date="2021-02-21">
     <ul>
       <li>Add accelerator Ctrl+D for insert timestamp</li>
       <li>WebDAV sync no longer uses wdfs</li>
@@ -60,7 +60,7 @@
       <li>Fix crash when synchronizing immediately after configuring</li>
     </ul>
   </release>
-  <release version="40.alpha" date="2021-01-10">
+  <release version="40~alpha" date="2021-01-10">
     <ul>
       <li>Change shortcut for find first/next to Ctrl-G/Ctrl-Shitf-G</li>
       <li>Change icons and remove labels from find next/previous</li>
