diff --git a/data/org.gnome.Gnote.appdata.xml.in b/data/org.gnome.Gnote.appdata.xml.in
index 3450c956..f7a52fca 100644
--- a/data/org.gnome.Gnote.appdata.xml.in
+++ b/data/org.gnome.Gnote.appdata.xml.in
@@ -14,9 +14,9 @@
   <p>Gnote also supports synchronization, making it simple to use it on multiple devices.</p>
  </description>
  <screenshots>
-  <screenshot type="default" width="790" height="516">https://wiki.gnome.org/Apps/Gnote?action=AttachFile&amp;do=get&amp;target=gnote_main_window.png</screenshot>
-  <screenshot type="default" width="880" height="633">https://wiki.gnome.org/Apps/Gnote?action=AttachFile&amp;do=get&amp;target=start_here.png</screenshot>
-  <screenshot type="default" width="880" height="633">https://wiki.gnome.org/Apps/Gnote?action=AttachFile&amp;do=get&amp;target=using_links.png</screenshot>
+  <screenshot type="default">https://wiki.gnome.org/Apps/Gnote?action=AttachFile&amp;do=get&amp;target=gnote_main_window.png</screenshot>
+  <screenshot type="default">https://wiki.gnome.org/Apps/Gnote?action=AttachFile&amp;do=get&amp;target=start_here.png</screenshot>
+  <screenshot type="default">https://wiki.gnome.org/Apps/Gnote?action=AttachFile&amp;do=get&amp;target=using_links.png</screenshot>
  </screenshots>
  <url type="homepage">https://wiki.gnome.org/Gnote</url>
  <url type="bugtracker">https://gitlab.gnome.org/GNOME/gnote/issues</url>
